src/components/PasswordDialog.vue

```diff
@@ -66,18 +67,31 @@ export default defineComponent({
 		return {
 			password: '',
 			loading: false,
-			showError: false,
+			error: false as boolean|number,
 		}
 	},
 
 	computed: {
 		helperText() {
-			if (this.showError) {
-				return this.password === '' ? t('Please enter your password') : t('Wrong password')
+			if (this.error !== false) {
+				if (this.password === '') {
+					return t('Please enter your password')
+				}
+
+				switch (this.error) {
+					case true:
+						return t('Unknown error while checking password')
+					case 403:
+						return t('Wrong password')
+					default:
+						return t('Error {requestStatus} while checking password', { requestStatus: this.error })
```

Comment:
But this is also just a "unknown error" showing users the http status code will very likely just confuse them, no?

Reply:
Agree. As a user, I don't know what I should do when it says "Error 502".